tests/integration_test/services_test.py

```
@@ -1479,8 +1479,8 @@ def test_secure_collection_crud_lifecycle():
         },
         "status": True
     }
-    add_resp = client.post(
-        f"/api/bot/{bot_id}/collections/data/add",
+    add_resp_1 = client.post(
```

Ruff (0.11.9)

Local variable `add_resp_1` is assigned to but never used

Remove assignment to unused variable `add_resp_1`

(F841)